### ISSUE TITLE
ff-data-table suggestion: do not show sort icon if no sort applied on that column

### ISSUE THREAD

**Issue by knolleary** [flowforge/forge-ui-components#39]
### Description

Whilst reviewing the migration to `ff-data-table`, the one bit that sticks out to me is the sort icon. When there are lots of columns, there are lots of sort icons - which adds noise.

<img width="550" alt="image" src="https://user-images.githubusercontent.com/51083/182174848-4a45ea2c-c575-4357-b6e1-f9df7ac16872.png">

An enhancement I'd like to be considered is:
 - hide the sort icon if the column doesn't have a sort currently applied
 - show the sort icon when the mouse hovers the header to give the cue that it can be sorted

**Comment by joepavitt** [comment]
I had actually designed that as per your request the first time round, but I found it unclear to the user that sort is an available option, especially because we do not use vertical dividers. It also meant that the center aligned columns (e.g. the 4, 1, 0 in this case) look better, otherwise they're center aligned to a column, but the header text is left-aligned.

I can look at reducing the size of the icons (as I agree they're noisy).
